package.json

```
@@ -12,30 +12,34 @@
     "format": "prettier --write './**/*.{js,jsx,ts,tsx,css,scss,md,mdx}'",
     "prepare": "husky install",
     "hide": "git secret hide -m",
-    "reveal": "git secret reveal -f"
+    "reveal": "git secret reveal -f",
+    "test:unit": "vitest --run && vitest run --coverage"
```

```suggestion
    "test:unit": "vitest run --coverage"
```

`vitest run --coverage` でテストも実行されてます。